### PR TITLE
Dev/#362 adding reply to header to email form

### PR DESCRIPTION
**Pull request by mcmpp**

### CODE REVIEW

File: src/main/scala/ch/openolitor/arbeitseinsatz/eventsourcing/ArbeitseinsatzEventStoreSerializer.scala [hunk 1]
```diff
@@ -26,14 +26,21 @@ import ch.openolitor.arbeitseinsatz._
...
 import ch.openolitor.core.domain.EntityStoreJsonProtocol
-import ch.openolitor.stammdaten.models.PersonContactPermissionModify
+import ch.openolitor.core.eventsourcing.CoreEventStoreSerializer
+import ch.openolitor.mailtemplates.eventsourcing.MailTemplateEventStoreSerializer
+import ch.openolitor.stammdaten.StammdatenJsonProtocol
+import ch.openolitor.stammdaten.models.{ Person, PersonContactPermissionModify }
+import spray.json.JsValue
 import spray.json.lenses.JsonLenses._
-import stamina.{ V1, V2 }
+import stamina.{ V1, V2, V3 }
```
Comment (toggm) [inline src/main/scala/ch/openolitor/arbeitseinsatz/eventsourcing/ArbeitseinsatzEventStoreSerializer.scala]: I guess you should run organize imports to reduce number of warnings?

File: src/main/scala/ch/openolitor/arbeitseinsatz/eventsourcing/ArbeitseinsatzEventStoreSerializer.scala [hunk 2]
```diff
@@ -26,14 +26,21 @@ import ch.openolitor.arbeitseinsatz._
...
 import stamina.json._
 import zangelo.spray.json.AutoProductFormats
+import spray.json.lenses.JsonLenses._
 
-trait ArbeitseinsatzEventStoreSerializer extends ArbeitseinsatzJsonProtocol with EntityStoreJsonProtocol with AutoProductFormats[JSONSerializable] {
+trait ArbeitseinsatzEventStoreSerializer extends ArbeitseinsatzJsonProtocol with EntityStoreJsonProtocol with CoreEventStoreSerializer with AutoProductFormats[JSONSerializable] {
+  val False = false
```
Comment (toggm) [inline src/main/scala/ch/openolitor/arbeitseinsatz/eventsourcing/ArbeitseinsatzEventStoreSerializer.scala]: Really needed?

File: src/main/scala/ch/openolitor/arbeitseinsatz/ArbeitseinsatzAktionenService.scala [hunk 3]
```diff
@@ -55,22 +56,23 @@ class ArbeitseinsatzAktionenService(override val sysConfig: SystemConfig, overri
   with ArbeitseinsatzDBMappings with MailServiceReference with ArbeitseinsatzEventStoreSerializer {
   self: ArbeitseinsatzWriteRepositoryComponent =>
 
+  override val False = false
```
Comment (toggm) [inline src/main/scala/ch/openolitor/arbeitseinsatz/ArbeitseinsatzAktionenService.scala]: Really needed?

File: src/main/scala/ch/openolitor/buchhaltung/eventsourcing/BuchhaltungEventStoreSerializer.scala [hunk 4]
```diff
@@ -32,10 +32,12 @@ import ch.openolitor.buchhaltung.BuchhaltungCommandHandler._
 import zangelo.spray.json.AutoProductFormats
 import ch.openolitor.core.eventsourcing.CoreEventStoreSerializer
 import ch.openolitor.core.JSONSerializable
+import spray.json.lenses.JsonLenses._
 
 trait BuchhaltungEventStoreSerializer extends BuchhaltungJsonProtocol with EntityStoreJsonProtocol with CoreEventStoreSerializer with AutoProductFormats[JSONSerializable] {
   import ch.openolitor.core.eventsourcing.events._
 
+  val False = false;
```
Comment (toggm) [inline src/main/scala/ch/openolitor/buchhaltung/eventsourcing/BuchhaltungEventStoreSerializer.scala]: same same

File: src/main/scala/ch/openolitor/core/eventsourcing/EventStoreSerializer.scala [hunk 5]
```diff
@@ -56,6 +56,7 @@ object EventStoreSerializer extends EntityStoreJsonProtocol
   with CoreEventStoreSerializer
   with SystemEventSerializer {
 
+  override val False = false
```
Comment (toggm) [inline src/main/scala/ch/openolitor/core/eventsourcing/EventStoreSerializer.scala]: same same

Comment (mcmpp) [reply src/main/scala/ch/openolitor/core/eventsourcing/EventStoreSerializer.scala]: done

File: src/main/scala/ch/openolitor/buchhaltung/eventsourcing/BuchhaltungEventStoreSerializer.scala (hunk 4 above)
Comment (mcmpp) [reply src/main/scala/ch/openolitor/buchhaltung/eventsourcing/BuchhaltungEventStoreSerializer.scala]: done

File: src/main/scala/ch/openolitor/arbeitseinsatz/ArbeitseinsatzAktionenService.scala (hunk 3 above)
Comment (mcmpp) [reply src/main/scala/ch/openolitor/arbeitseinsatz/ArbeitseinsatzAktionenService.scala]: done

File: src/main/scala/ch/openolitor/arbeitseinsatz/eventsourcing/ArbeitseinsatzEventStoreSerializer.scala (hunk 2 above)
Comment (mcmpp) [reply src/main/scala/ch/openolitor/arbeitseinsatz/eventsourcing/ArbeitseinsatzEventStoreSerializer.scala]: done

**Review by toggm — APPROVED** [review]

File: src/main/scala/ch/openolitor/arbeitseinsatz/eventsourcing/ArbeitseinsatzEventStoreSerializer.scala (hunk 1 above)
Comment (mcmpp) [reply src/main/scala/ch/openolitor/arbeitseinsatz/eventsourcing/ArbeitseinsatzEventStoreSerializer.scala]: I have removed the ones that are not used right now. If you have added the "organize import" in your otp story, we can integrate it when merging these branch

**Pull request merged**